### DISCUSSION TITLE
Import Posts from Wordpress

### DISCUSSION THREAD

**Discussion by otatechie** [statamic/cms#5655] [Q&A]
What is the best way to import posts from WordPress?

**Comment by bfaulkner-champions** [comment]
I used the [Statamic WP Import addon](https://statamic.com/addons/stopa-development/statamic-wp-import). I did have some small issues, which were:

- Need to make sure collections exist in advance.
- Same for taxonomies.
- Errors tended to result in a 500 error rather than be gracefully handled.
- URLs need to be converted before import, which isn't a huge bother.

That said, it was still a massive help.

Otherwise there's a package called Corcel which I've not used but is claimed to give more control. The article below is a little outdated but seems otherwise to have good feedback. 
[https://ghijk.co.uk/blog/import-posts-from-wordpress-to-statamic](https://ghijk.co.uk/blog/import-posts-from-wordpress-to-statamic)
